nodes/mtdevice.py

```diff
@@ -1254,6 +1258,85 @@ def usage():
             For longitude, latitude, altitude and orientation (on MTi-G-700):
                 "pl400fe,pa400fe,oq400fe"
 
+Synchronization settings:
+    The format follows the xsens protocol documentation:
+    |Function| Line| Trigger Type| Skip First|...
+    Skip Factor| Pulse Width| Delay or Clock period or offset|
+    It is required to have all fields presenst in the settings argument
+    Note: The entire synchronization buffer is wiped every time a new one 
+          is set, so it is necessary to specify the settings of multiple 
+          lines at once.
+
+        Function (see manual for details):
+            3 Trigger indication
+            4 Interval Transition Measurement
+            8 SendLatest
+            9 ClockBiasEstimation
+            11 StartSampling
```

Comment:
Ok, I was not clear, the two spaces I was talking about are between the letter/number and its description:
`11__StartSampling` instead of `11_StartSampling` (with space instead of underscore, of course).